frontend/public/components/dashboard/dashboards-page/cluster-dashboard/cluster-dashboard.tsx

```diff
@@ -19,24 +19,22 @@ const mainCards = [{ Card: StatusCard }, { Card: UtilizationCard }];
 const leftCards = [{ Card: DetailsCard }, { Card: InventoryCard }];
 const rightCards = [{ Card: ActivityCard }];
 
+const HIDE_QUICK_START_DASHBOARD_TILE_USER_SETTINGS_KEY = 'console.dashboard.quickStartTile';
+
 export const ClusterDashboard: React.FC<{}> = () => {
-  const isQuickStartsCatalogCardHidden =
-    localStorage.getItem(HIDE_QUICK_START_DASHBOARD_TILE_STORAGE_KEY) === 'true';
   const [infrastructure, infrastructureLoaded, infrastructureError] = useK8sGet<K8sResourceKind>(
     InfrastructureModel,
     'cluster',
   );
-  const [showQuickStartsCatalogCard, setShowQuickStartsCatalogCard] = React.useState<boolean>(
-    !isQuickStartsCatalogCardHidden,
+  // [TODO](sahil143): use sync capability here
+  const [showQuickStartsCatalogCard, , loaded] = useUserSettings(
+    HIDE_QUICK_START_DASHBOARD_TILE_USER_SETTINGS_KEY,
+    true,
   );
 
-  const onQuickStartsCatalogCardRemove = () => {
```

Comment:
@sahil143 when the quick start card is removed, without this it leaves a gap above the Activity card as in the gird doesn't re-arrange itself properly

Reply:
Yeah, This should be fixed once the sync capability is available. userSettings will update the values once the quickStart card is removed.

Reply:
This is a one-line change which can go in with sync PR as well